src/main/java/com/scottlogic/filters/AuthorPostFilter.java

```diff
@@ -0,0 +1,28 @@
+package com.scottlogic.filters;
+
+import com.scottlogic.PostFilter;
+import com.scottlogic.UserPost;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class AuthorPostFilter implements PostFilter {
+
+    final String Author;
```

Comment:
`final` here doesn't set a scope this means that I think (though I could be wrong) that this is a protected scoped variable which means that it can be access within the same module. In general it is preferred to put a `private` as well as a `final` to make the scoping explicit to save confusion.